### PR TITLE
Fix editing tags / icon (setting subelement text)

### LINKED ISSUE #56: Can't modify or create entry tags
`entry.tags = ['some', 'thing']` doesn't work. Tags stay the same. I think the error is in the `BaseElement._set_subelement_text` tried to fix it but no luck.

### PR DESCRIPTION
**Pull request by gpchelkin**
Remove subelement and create it with passed value. Fix #56

### CODE REVIEW

**Comment by Evidlo** [comment]
Could you add a test to EntryTests for this? Maybe `test_set_fields`.  We should probably test setting all attributes there.

*edit* moved comment to PR

File: tests/tests.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from datetime import datetime
+from datetime import datetime,
```
Comment (pschmitt) [inline tests/tests.py]: `datetime,` ?

Comment (gpchelkin) [reply tests/tests.py]: already fixed. yeah, I am watching travis logs too and trying to pass 😄

**Comment by pschmitt** [comment]
LGTM

**Pull request merged**